hbase-common/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileContext.java

```diff
@@ -203,11 +218,18 @@ public String getHFileName() {
   public long heapSize() {
     long size = ClassSize.align(ClassSize.OBJECT +
         // Algorithm reference, encodingon, checksumtype, Encryption.Context reference
-        5 * ClassSize.REFERENCE +
+        7 * ClassSize.REFERENCE +
```

Comment:
Seems this is the only difference when cherry-picking? I'd rather using the same expression as HBASE-23251 if possible so future change would find comfortable eyeballs.

Reply:
@liuml07 - the change to heap size was required to get an existing test in TestHFileBlock to pass.

Reply:
Thanks @gjacoby126 I guess my comment is more about format for easier review and future change.
```
5 * ClassSize.REFERENCE +
2 * Bytes.SIZEOF_INT +
// usesHBaseChecksum, includesMvcc, includesTags and compressTags
4 * Bytes.SIZEOF_BOOLEAN +
Bytes.SIZEOF_LONG);
```
to
```
7 * ClassSize.REFERENCE +
2 * Bytes.SIZEOF_INT +
// usesHBaseChecksum, includesMvcc, includesTags and compressTags
4 * Bytes.SIZEOF_BOOLEAN +
2 * ClassSize.ARRAY +
Bytes.SIZEOF_LONG);
```

Instead, I think it's better to have the same expression as `master` branch.
```
// Algorithm, checksumType, encoding, Encryption.Context, hfileName reference,
5 * ClassSize.REFERENCE + 2 * Bytes.SIZEOF_INT +
// usesHBaseChecksum, includesMvcc, includesTags and compressTags
4 * Bytes.SIZEOF_BOOLEAN + Bytes.SIZEOF_LONG +
//byte[] headers for column family and table name
2 * ClassSize.ARRAY + 2 * ClassSize.REFERENCE);
```

Reply:
done